src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -724,6 +743,10 @@ void prepare_primitive_fusing::fuse_simple_primitives(program &p) {
         };
 
         auto fuse_activation_f = [&](activation_node& activation_node) {
+            GPU_DEBUG_IF(p.get_config().get_disable_post_ops_fusions() != 0) {
+                GPU_DEBUG_IF(p.get_config().get_disable_post_ops_fusions() != 11)
```

Comment:
I think this way can't support activation, quantize and eltwise several  combinations for debugging. In real-world use, I needed this combination. How about change using bit mask? For example

> constexpr uint32_t POST_OP_QUANTIZE   = 1 << 1; // 0b0010
> constexpr uint32_t POST_OP_ELTWISE    = 1 << 2; // 0b0100
> 
> GPU_DEBUG_IF(p.get_config().get_disable_post_ops_fusions() & POST_OP_ACTIVATION) {
>     return; // skip activation fusion
> }

Reply:
Disscussed online.